.travis.yml

```diff
@@ -5,6 +5,10 @@ services:
   - docker
 language: go
 go_import_path: go.uber.org/yarpc
+branches:
+  only:
+    - master
+    - dev
```

Comment:
This means that we can never run tests on stacked diffs ;_;

Can we revert this?